<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(69, 4, 3);
font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelPlainText">
          <property name="styleSheet">
           <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Plain Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelKey">
          <property name="styleSheet">
           <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEncryptedText">
          <property name="styleSheet">
           <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Encrypted Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDecryptedText">
          <property name="styleSheet">
           <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Decrypted Text</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditPlainText">
          <property name="styleSheet">
           <string notr="true">border-radius:2px;
background-color:rgb(88, 111, 124);
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditKey">
          <property name="styleSheet">
           <string notr="true">border-radius:2px;
background-color:rgb(88, 111, 124);
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="placeholderText">
           <string>Minumum key lenght is 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditEncryptedText">
          <property name="styleSheet">
           <string notr="true">border-radius:2px;
background-color:rgb(88, 111, 124);
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDecryptedText">
          <property name="styleSheet">
           <string notr="true">border-radius:2px;
background-color:rgb(88, 111, 124);
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonEncrypt">
            <property name="text">
             <string>ENCRYPT</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonDecrypt">
            <property name="text">
             <string>DECRYPT             </string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRUN">
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius:2px;
background-color:rgb(88, 111, 124);
color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color:rgb(88, 181, 124);
}
QPushButton:pressed{
background-color:rgb(118, 131, 124);
}</string>
            </property>
            <property name="text">
             <string>RUN</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditError">
          <property name="styleSheet">
           <string notr="true">border-radius:2px;
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 55, 55);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="labelSanitizedText">
              <property name="styleSheet">
               <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Sanitized Text</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSanitizedKey">
              <property name="styleSheet">
               <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Sanitized Key</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>117</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLineEdit" name="lineEditSanitizedText">
              <property name="styleSheet">
               <string notr="true">border-radius:2px;
background-color:rgb(88, 111, 124);
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSanitizedKey">
              <property name="styleSheet">
               <string notr="true">border-radius:2px;
background-color:rgb(88, 111, 124);
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelPlainTextCipherTable">
              <property name="styleSheet">
               <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Cipher Table</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton5x5">
              <property name="text">
               <string>5x5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton6x6">
              <property name="text">
               <string>6x6</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="labelPlainText_4">
            <property name="styleSheet">
             <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Table Seed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTableSeed">
            <property name="styleSheet">
             <string notr="true">border-radius:2px;
background-color:rgb(88, 111, 124);
font:87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetCipherTable">
          <property name="styleSheet">
           <string notr="true">QHeaderView::section {
	background-color:rgb(88, 111, 124);
    color: white; 
    font-weight: bold;
}
QHeaderView::section:horizontal {
	height: 18px

}
QHeaderView::section {
padding: 4px;
}
QTableWidget{
background-color: rgb(69, 4, 3);
font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>68</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>A</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>D</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>F</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>G</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>V</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>X</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>A</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>D</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>F</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>G</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>V</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="labelTranspositionTable">
          <property name="text">
           <string>Transposition Table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetTranspositionTable">
          <property name="styleSheet">
           <string notr="true">QHeaderView::section {
	background-color:rgb(88, 111, 124);
    color: white; 
    font-weight: bold;
}
QHeaderView::section:horizontal {
	height: 18px

}
QHeaderView::section {
padding: 4px;
}
QTableWidget{
background-color: rgb(69, 4, 3);
font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>68</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="labelTranspositionTableAlphabetically">
          <property name="text">
           <string>Transposition Table Alphabetically</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetTranspositionTableAlphabetically">
          <property name="styleSheet">
           <string notr="true">QHeaderView::section {
	background-color:rgb(88, 111, 124);
    color: white; 
    font-weight: bold;
}
QHeaderView::section:horizontal {
	height: 18px

}
QHeaderView::section {
padding: 4px;
}
QTableWidget{
background-color: rgb(69, 4, 3);
font:87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>68</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
